Frontend/Application/.env

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_API_URL=https://api.trustlesswork.com
```

Comment:
This file should never be commited; please replace this with an `.env.example` with this contents:


```
NEXT_PUBLIC_API_URL=
```

And then for other developers, share this value in a secure environment and private manner.